mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -573,7 +573,16 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_5721 isTouchingWaterOrRain ()Z
 	METHOD method_5722 isTeammate (Lnet/minecraft/class_1297;)Z
 		ARG 1 other
-	METHOD method_5723 dealDamage (Lnet/minecraft/class_1309;Lnet/minecraft/class_1297;)V
+	METHOD method_5723 applyDamageEffects (Lnet/minecraft/class_1309;Lnet/minecraft/class_1297;)V
+		COMMENT Applies damage effects to {@code attacker} or {@code target}.
+		COMMENT
+		COMMENT <p>Called when {@code attacker} damages {@code target}.
```

Comment:
Notice the attacker is usually this entity, but may be a projectile's owner entity, etc. I would prefer clarifying that in the javadoc as well